java/ql/test/query-tests/SpuriousJavadocParam/Test.java

```diff
@@ -120,5 +120,16 @@ class GenericClass<T> {}
    */
   interface GenericInterface<T> {}
 
-  // Diagnostic Matches: Incomplete inheritance relation for type java.lang.Object and supertype none
+  /**
+   * @param i exists
+   * @param k does not
+   */
+  static record SomeRecord(int i, int j) {}
+
+  /**
+   * @param <T> exists
```

Comment:
Maybe also test non-existent type parameters.
```suggestion
   * @param <T> exists
   * @param <U> does not
```

Reply:
Added, but now you'll have to re-approve, I think.